### DISCUSSION TITLE
What is the name of the 🇯🇵 decluttering guru?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#974] [Galaxy Trivia] [answered]
**Bonus:**  What are her rules for 🧹🧼🪥?

**Comment by peterbaccaro** [comment, marked as answer]
The Japanese decluttering guru you are referring to is **Marie Kondo**. She gained international recognition with her best-selling book "The Life-Changing Magic of Tidying Up" and her unique decluttering and organizing method called the KonMari Method. Marie Kondo's approach focuses on keeping only items that spark joy and organizing belongings by category in a systematic way. Her ideas have resonated with people worldwide, and she has since become a well-known figure in the field of home organization and minimalism.

Marie Kondo's decluttering and organizing method is based on a set of principles and rules. Here are some of the key rules:

- **Rule of Discarding**: The first step is to discard items before organizing. Hold each item in your hands and ask yourself if it sparks joy. If it does, keep it. If not, thank it for its service and let it go.

- **Tidy by Category, Not Location**: Sort and organize items by category rather than tackling one room at a time. The recommended order is clothing, books, papers, miscellaneous items (komono), and sentimental items.

- **Vertical Folding**: Fold clothing items in a way that they can stand upright vertically, allowing you to see everything at a glance and maximize storage space.

- **Store Similar Items Together**: Group items of the same category together. For example, keep all books in one place or store all kitchen utensils in a single drawer.

- **Designate a Place for Each Item**: Assign a specific home for every item you own. This helps maintain organization and makes it easier to put things away.

- **Express Gratitude to Items**: Treat your belongings with respect and gratitude. Thank them for their service and appreciate their role in your life, even if you're letting go of them.

- **Focus on What to Keep, Not What to Discard**: Instead of fixating on what you want to get rid of, shift your mindset to focus on the items you want to keep and cherish.
